src/TwigComponent/CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
 # CHANGELOG
 
+## 2.5
+
+- Add new autowireable `ComponentRendererInterface`
```

Comment:
```suggestion
-   Add new autowireable `ComponentRendererInterface`
```

Let's keep the formatting consistent (I have issues with this formatting in my IDE too).